tools/quake3/q3map2/nav.cpp

```diff
@@ -44,10 +44,10 @@ struct Character
 static const Character characterArray[] = {
 	{ "builder",     20, 40 },
 //	{ "builderupg",  20, 40 },
-	{ "human_naked", 15, 56 },
+	{ "human_naked", 15, 40 },
```

Comment:
I would edit the commented out lines accordingly.

Reply:
Add a comment explaining why the crouch maxs are used

Reply:
https://www.youtube.com/watch?v=7EzOEwoMcSU